kuma/wiki/jinja2/wiki/includes/spam_error.html

```diff
@@ -1,19 +1,13 @@
 <p>
-    {% trans does_this_belong_url=wiki_url('MDN/Contribute/Does_this_belong') %}
-    Your submission includes content that may not be suitable for MDN.
-    Repeated attempts to submit unsuitable material may result in your
-    account being deactivated. See our guide
-    “<a href="{{ does_this_belong_url }}">Does this belong on MDN?</a>”
-    for details on what constitutes appropriate material.
-    {% endtrans %}
-</p>
-<p>
     {% trans admin_email='mailto:mdn-admins@mozilla.org' %}
-    If you believe you’ve received this message by mistake, please
-    save a draft of your content and <a href="{{ admin_email }}">
+    Sorry! We weren't able to save your contribution.
+    Please save a draft of your change and <a href="{{ admin_email }}">
     contact the site administrators</a>.
     {% endtrans %}
 </p>
+<p> We'll try to resolve the problem, and after that you will be able
+    to resubmit your contribution.
+</p>
```

Comment:
Please wrap the text inside the ``<p>`` tags in a ``{% trans %}`` / `` {% endtrans %} `` block, so that the text will be extracted for the localizers.  Alternatively, pull the sentence inside the early paragraph.